docs/sources/next/javascript-api/k6-experimental/browser/elementhandle/ishidden.md

```diff
@@ -0,0 +1,48 @@
+---
+title: 'isHidden()'
+excerpt: 'Browser module: elementHandle.isHidden() method'
+---
+
+# isHidden()
+
+Checks if the element is `hidden`.
+
+### Returns
+
+| Type | Description                                      |
+| ---- | ------------------------------------------------ |
+| bool | `true` if the element is `hidden`, else `false`. |
```

Comment:
Small suggestion, feel free to discard it if you think the current form is OK:
```suggestion
| bool | `true` if the element is `hidden`, `false` otherwise. |
```